.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+end_of_line = lf
```

Comment:
That might make sense for line endings, but as far as I know, `.gitattributes` will not automatically make your editor use two spaces versus four when you're typing, which `.editorconfig` will do.  I see this merely a convenience.   To enforce conventions like spacing, this repo really should be being linted and tested.